source/frontend/src/utils/mapPropertyUtils.ts

```diff
@@ -144,3 +147,59 @@ function toMapProperty(
     name: feature?.properties?.NAME,
   };
 }
+
+export function featuresetToMapProperty(
+  featureSet: LocationFeatureDataset,
+  address: string = 'unknown',
+): IMapProperty {
+  const pimsFeature = featureSet.pimsFeature;
```

Comment:
can featureset be null/undefined?

Reply:
Not at this point. The method signature guarantees is an object